backend/app/repositories/models/conversation.py

```diff
@@ -35,3 +35,8 @@ class ConversationMeta(BaseModel):
     create_time: float
     model: str
     bot_id: str | None
+
+
+class InvocationMetrics(BaseModel):
```

Comment:
We use "model" to handle CRUD with dynamodb. The `InvocationMetrics` is not for the purpose, so could you move it to bedrock.py?

Reply:
done